frontend/src/components/projectListView/ProjectListView.tsx

```diff
@@ -0,0 +1,252 @@
+import { Task } from '../../pages/projectView/ProjectView';
+import { useState, useEffect } from 'react';
+import ListTable from '../listTable/ListTable';
+import { AddCircleOutline } from '@mui/icons-material';
+import { TextField } from '@mui/material';
+import './projectList.css';
+import { v4 as uuid } from 'uuid';
+import { useParams } from 'react-router-dom';
+import { User, getAuth, onAuthStateChanged } from '@firebase/auth';
+import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
+
+const ProjectListView = ({ tasks }: { tasks: Task[] }) => {
+  const [allTasks, setAllTasks] = useState<Task[]>(tasks);
+
+  const [documentation, setDocumentation] = useState<Task[]>([]);
+  const [ongoing, setOngoing] = useState<Task[]>([]);
+  const [todo, setTodo] = useState<Task[]>([]);
+  const [done, setDone] = useState<Task[]>([]);
+
+  const [documentationDescription, setDocumentationDescription] =
+    useState<string>('');
+  const [ongoingDescription, setOngoingDescription] = useState<string>('');
+  const [todoDescription, setTodoDescription] = useState<string>('');
+  const [doneDescription, setDoneDescription] = useState<string>('');
+
+  const [documentationEditing, setDocumentationEditing] =
+    useState<boolean>(false);
+  const [ongoingEditing, setOngoingEditing] = useState<boolean>(false);
+  const [todoEditing, setTodoEditing] = useState<boolean>(false);
+  const [doneEditing, setDoneEditing] = useState<boolean>(false);
```

Comment:
Here is a lot of repetition, you can probably abstract this away in an object. Let's say you have an object representing a Category, that can have:
- An array of tasks
- A description
- Editing status

Abstracting it in this way will then also make it possible to remove/add categories as well open up for the ability for the user to create their own categories.